initials.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+int main()
```

Comment:
Nice! you used the wrong symbol though (#s, not $s), but not a big deal. Also, add comments!